app/src/main/java/org/sopt/dateroad/data/repositoryimpl/AuthRepositoryImpl.kt

```diff
@@ -55,7 +55,12 @@ class AuthRepositoryImpl @Inject constructor(
                 null
             } else {
                 ContentUriRequestBody(contentResolver = contentResolver, uri = Uri.parse(editProfile.image)).toFormData(name = PROFILE_FORM_DATA_IMAGE)
-            }
+            },
+            isDefaultImage = if (editProfile.image.isNullOrEmpty()) {
+                "true"
+            } else {
+                "false"
+            }.toRequestBody("application/json".toMediaType())
```

Comment:
```suggestion
isDefaultImage = Json.encodeToString(if (editProfile.image.isNullOrEmpty()) {
                true
            } else {
                false
            }).toRequestBody("application/json".toMediaType())
```

일케 하면 "true" 이런 거 안 써두 되지롱
근데 여기 if문 너무 많아서 복잡해지는 것 같아서 고민 좀 해보겠슴다
오늘 너무 바빠서 낼 할게오 ㅠㅠ,,